<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="trainButton">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
            <weight>62</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(62, 162, 255);
border: none;
color: &quot;white&quot;;
padding-bottom: 10px;
padding-top: 10px;
border-radius: 9;
font-weight: 500;
font-size: 16px;</string>
          </property>
          <property name="text">
           <string>Trenuj</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="examButton">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(62, 162, 255);
border: none;
color: &quot;white&quot;;
padding-bottom: 10px;
padding-top: 10px;
border-radius: 9;
font-weight: 500;
font-size: 16px;</string>
          </property>
          <property name="text">
           <string>Test</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="statsButton">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(62, 162, 255);
border: none;
color: &quot;white&quot;;
padding-bottom: 10px;
padding-top: 10px;
border-radius: 9;
font-weight: 500;
font-size: 16px</string>
          </property>
          <property name="text">
           <string>Statystyki</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="pagesStack">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <pointsize>16</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(77, 77, 77);
</string>
              </property>
              <property name="text">
               <string>Kategorie</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="categoriesField">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">QListWidget
{
	border : 2px solid rgb(62, 162, 255);
	background-color: 'white';
}
QListWidget QScrollBar
{
	background : lightblue;
}
QListView::item:selected
{
	background-color: rgb(222, 222, 222);
}
QListView::item
{
	padding-top: 7px;
	padding-bottom: 7px;
   color: rgb(77, 77, 77);
}
QListView
{
	outline: 0;
   font-size: 16px;
   font-weight: 450;
}</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QWidget" name="widget" native="true">
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="choosenCategoryHeader">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Wybrana kategoria:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="cardName">
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>Airplane</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="answerFeedbackLabel">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>70</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>-1</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">padding: 10px;
font-size: 18px;
font-weight: 400;</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="flashcardsInputsContainer" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="baseSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLineEdit" name="answerInput"/>
                </item>
                <item>
                 <widget class="QPushButton" name="flashcardButton">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Sprawdź</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QPushButton" name="testEasyBtn">
               <property name="styleSheet">
                <string notr="true">color: rgb(62, 162, 255);
border-radius: 5px;
background-color: 'white';
border: 2px solid rgb(62, 162, 255);
padding: 7px;
padding-left: 15px;
padding-right: 15px;
font-weight: 500;</string>
               </property>
               <property name="text">
                <string>Łatwy</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="testMediumBtn">
               <property name="styleSheet">
                <string notr="true">color: rgb(62, 162, 255);
border-radius: 5px;
background-color: 'white';
border: 2px solid rgb(62, 162, 255);
padding: 7px;
padding-left: 15px;
padding-right: 15px;
font-weight: 500;</string>
               </property>
               <property name="text">
                <string>Średni</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="testHardBtn">
               <property name="styleSheet">
                <string notr="true">color: rgb(62, 162, 255);
border-radius: 5px;
background-color: 'white';
border: 2px solid rgb(62, 162, 255);
padding: 7px;
padding-left: 15px;
padding-right: 15px;
font-weight: 500;</string>
               </property>
               <property name="text">
                <string>Trudny</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>118</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="testStartBtn">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton:enabled {
color: rgb(62, 162, 255);
border-radius: 5px;
background-color: 'white';
border: 2px solid rgb(62, 162, 255);
padding: 7px;
padding-left: 15px;
padding-right: 15px;
font-weight: 500;
}
QPushButton:disabled {
color: rgb(62, 162, 255);
border-radius: 5px;
background-color: 'white';
border: 2px solid rgb(62, 162, 255);
padding: 7px;
padding-left: 15px;
padding-right: 15px;
font-weight: 500;
border: 2px solid rgb(118, 152, 184);
color: rgb(118, 152, 184);
background-color: rgb(227, 227, 227);
}</string>
               </property>
               <property name="text">
                <string>Rozpocznij</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QStackedWidget" name="testStack">
            <property name="currentIndex">
             <number>1</number>
            </property>
            <widget class="QWidget" name="page_6">
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QWidget" name="testsMainDisplay" native="true">
                <layout class="QVBoxLayout" name="verticalLayout_7">
                 <item>
                  <widget class="QLabel" name="testCardLabel">
                   <property name="text">
                    <string>TextLabel</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QWidget" name="widget_4" native="true">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout_7">
                    <item>
                     <widget class="QLineEdit" name="testCardInput"/>
                    </item>
                    <item>
                     <widget class="QPushButton" name="testCardBtn">
                      <property name="minimumSize">
                       <size>
                        <width>150</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>Dalej</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_7"/>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_3">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="styleSheet">
               <string notr="true">color: rgb(62, 162, 255);
border-radius: 5px;
background-color: 'white';
border: 2px solid rgb(62, 162, 255);
padding: 5px;
font-weight: 500;</string>
              </property>
              <property name="text">
               <string>Odpowiedzi</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_2">
              <property name="styleSheet">
               <string notr="true">color: rgb(62, 162, 255);
border-radius: 5px;
background-color: 'white';
border: 2px solid rgb(62, 162, 255);
padding: 5px;
font-weight: 500;</string>
              </property>
              <property name="text">
               <string>Testy</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_3">
              <property name="styleSheet">
               <string notr="true">color: rgb(62, 162, 255);
border-radius: 5px;
background-color: 'white';
border: 2px solid rgb(62, 162, 255);
padding: 5px;
font-weight: 500;</string>
              </property>
              <property name="text">
               <string>Inne</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QStackedWidget" name="stackedWidget">
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <widget class="QWidget" name="page_4">
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QWidget" name="correctPlot" native="true"/>
              </item>
              <item>
               <widget class="QWidget" name="wrongPlot" native="true"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_5"/>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>465</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
